appinventor/blocklyeditor/src/blocks/text.js

```diff
@@ -317,21 +317,126 @@ Blockly.Blocks['text_starts_at'] = {
 };
 
 Blockly.Blocks['text_contains'] = {
-  // Is text contained in
   category: 'Text',
-  helpUrl: Blockly.Msg.LANG_TEXT_CONTAINS_HELPURL,
+
+  helpUrl: function() {
+    var mode = this.getFieldValue('OP');
```

Comment:
Minor refactor: add a `getMode()` method and use it anywhere you currently create `var mode`.